sauce/features/general/hide-help/index.js

```diff
@@ -0,0 +1,61 @@
+import { Feature } from 'core/feature';
+
+export class hideHelp extends Feature {
+
+  injectCSS() { return require('./index.css'); }
+
+  shouldInvoke() {
+    if (this.settings.enabled === true) {
+      return true;
+    }
+  }
+
+  observe(changedNodes) {
+    if (!this.shouldInvoke()) return;
+
+    if (changedNodes.has('ynab-u modal-popup modal-user-prefs ember-view modal-overlay active')) {
+      this.invoke();
+    }
+  }
+
+  invoke() {
+    let hide = ynabToolKit.shared.getToolkitStorageKey('hide-help', 'boolean');
```

Comment:
if you `import * as toolkitHelper from 'helpers/toolkit'` you can use the "sauce" version of this helper.

`toolkitHelper.getToolkitStorageKey()`